frontend/src/components/layout/topmessage/PhoneSurvey.module.scss

```diff
@@ -0,0 +1,47 @@
+@import "../../../styles/tokens.scss";
```

Comment:
Note: this file is a complete copy-paste from `InterviewRecruitment.module.scss`, and `PhoneSurvey.tsx` is basically `InterviewRecruitment.tsx` with a couple of renames. I didn't have a strong preference for adjusting the existing component and changing when it was displayed, or adding a new one - I went with the latter, but happy to change to the former.